pkgs/development/python-modules/pyrevolve/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, contexttimer
+, versioneer
+, cython
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "pyrevolve";
+  version = "2.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b49aea5cd6c520ac5fcd1d25fa23fe2c5502741d2965f3eee10be067e7b0efb4";
+  };
+
+  buildInputs = [ versioneer cython numpy ];
```

Comment:
cython and versioneer go into nativeBuildInputs. numpy is not used.

Reply:
I still have difficulty differentiating nativeBuildInputs and buildInputs.

Reply:
nativeBuildInputs is everything that is run on the build machine. buildInputs is everything that is linked against and used on the final machine. This is required for cross compiling.